ui/src/parts/lightbox.tsx

```diff
@@ -22,93 +22,83 @@ import { getColorFromString, isDark } from '@fluentui/react'
 export const lightboxB: Box<LightboxProps | null> = box()
 
 const
-  HEADER_HEIGHT = 40,
-  IMAGE_CAPTIONS_HEIGHT = 40, // Total height of title and decription.
-  FOOTER_VERTICAL_PADDING = 10,
-  IMAGE_NAV_HEIGHT = 148,
-  IMAGE_SIZE = 120,
-  ICON_SIZE = 38
+  IMAGE_CAPTIONS_HEIGHT = 46, // Total height of title and decription.
+  NAV_IMAGE_SIZE = 120,
+  IMAGE_NAV_HEIGHT = NAV_IMAGE_SIZE + 28, // 28 is the height of the scrollbar.
+  ICON_SIZE = 38,
+  ICON_MARGIN = 8,
+  IMAGE_NAV_BOTTOM_MARGIN = 8
 
 const
   css = stylesheet({
     body: {
       position: 'fixed',
       inset: '0px',
       zIndex: 1,
```

Comment:
Is this `z-index` needed?

Reply:
It is necessary when you have the image on the same page together with other components with `zIndex: 1` specified, e.g. the `ui.header`:
![image](https://user-images.githubusercontent.com/23740173/203530849-e35fe9ed-dcff-4d2c-afb1-4d67c53e8122.png)

It is maybe good to consider if the `zIndex` with value of `1` is enough and shouldn't be increased just in case.

Reply:
I see. I don't see a reason why `ui.header` should have special `z-index`, but it may be a valid option in some other cases.

Reply:
Yes, keep it.